Makefile

```diff
@@ -97,34 +82,33 @@ all: scratch check install
 # ============
 # DEPENDENCIES
 # ============
-
-$(GOVERALLS_BIN):
-	@$(call print, "Fetching goveralls.")
-	go get -u $(GOVERALLS_PKG)
-
 $(LINT_BIN):
-	@$(call print, "Fetching linter")
-	$(DEPGET) $(LINT_PKG)@$(LINT_COMMIT)
+	@$(call print, "Installing linter.")
+	go install $(LINT_PKG)
```

Comment:
Why not replace all these go installs into something like https://github.com/ossf/scorecard/blob/f9b9773e2f4632b48b21765413f069504be40714/Makefile#L41

Reply:
That's a cool idea. And if we only used the tools in the CI pipeline I'd say we add and use that single command. But since we also use the tools locally, it doesn't make sense to build them all when you just need one binary. Also the `$(LINT_BIN):` syntax causes `make` to skip a target if that file already exists.
But I like the idea of having all tools in a separate gomodule to remove those dependencies from the main module. I'm going to experiment with that in a separate PR.